### PR TITLE
Update platform installation documentation

### PR DESCRIPTION
**Pull request by guillaumeeb**
With Elastic Kubernetes last information.

With Worker memory limitations discussion.

### CODE REVIEW

File: EGI.md [hunk 1]
```diff
@@ -123,7 +125,8 @@ Out of all the configuration options, please pay special attention to the follow
        must be set to the one configure in [Step 1](#step-1-dns-name).
 * `Elastic Data` tab:
   * `Maximum Number of WNs in the cluster`: please set the maximum of node you'll want the 
-    Kubernetes cluster to expand to. Minimum cannot be set, it is positionned to 0 in the Tosca template.
+    Kubernetes cluster to expand to. 
+  * `Min Number of free WNs in the cluster`: 0 or 1, not sure about the meaning of `free`.
```
Comment (sebastian-luna-valero) [inline EGI.md:129]: Not sure I understand the meaning of `free` here? Here is what I see:
image

**Review by sebastian-luna-valero — APPROVED** [review]
Thanks!

File: EGI.md (hunk 1 above)
Comment (guillaumeeb) [reply EGI.md:129]: What is a _free_ WNs ? A WN withtou any pod? A WN which has still some resources? I don't understand how this min number value will be used in the end.

Comment (sebastian-luna-valero) [reply EGI.md:129]: Oh sorry, re-reading this properly now.

`free` means the number of WNs without allocated pods in the cluster that you would like to have.

Creating new WNs comes with its own overhead. With this option you ensure the number of empty WNs in preparation for a peak workload, and therefore you save some time when growing the cluster.

Comment (guillaumeeb) [reply EGI.md:129]: Thanks, this is clear now! So I guess we'll want to set that to 0 or 1, not more.

**Pull request merged**